.github/workflows/main.yml

```diff
@@ -74,6 +74,8 @@ jobs:
     - name: Test packages
       run: just test-packages
 
+    - name: Test postCreate script
+      run: just test-postcreate-script
```

Comment:
This is incredibly picky, but could you add a newline after this command, please? It's hard to see the publish line after it otherwise.

Reply:
Oh sure, not at all picky good spotting!